pleskdistup/actions/common.py

```diff
@@ -277,3 +277,44 @@ def _post_action(self) -> action.ActionResult:
 
     def _revert_action(self) -> action.ActionResult:
         return action.ActionResult()
+
+
+class RevertChangesInGrub(action.ActiveAction):
+    grub_configs_paths: typing.List[str]
+
+    def __init__(self):
+        self.name = "revert changes in grub made by elivate"
+        self.grub_configs_paths = [
+            "/boot/grub2/grub.cfg",
```

Comment:
Are these all the possible paths? E.g. I see `/boot/grub` instead of `/boot/grub2` on Ubuntu 22.04.

Reply:
Sure, feel free to propose other paths as well